packages/mdc-textfield/_mixins.scss

```diff
@@ -589,13 +581,9 @@
 }
 
 @mixin mdc-text-field-hover-outline-color_($color) {
-  &:not(.mdc-text-field--focused) {
-    // stylelint-disable-next-line selector-combinator-space-after
-    .mdc-text-field__input:hover ~,
-    .mdc-text-field__icon:hover ~ {
-      .mdc-notched-outline {
-        @include mdc-notched-outline-color($color);
-      }
+  &:not(.mdc-text-field--focused):hover {
+    .mdc-notched-outline {
+      @include mdc-notched-outline-color($color);
```

Comment:
Why did these lines change?

Reply:
I can revert it, it's being fixed in the [line ripple changes](https://github.com/material-components/material-components-web/pull/5366/files#diff-602886a88f66102525d0b2eb6eed3479L591). I think I originally fixed it here too because my first draft removed the `__icon` class entirely.